modules/apps/27-interchain-accounts/controller/client/cli/tx.go

```diff
@@ -46,13 +49,23 @@ the associated capability.`),
 				return err
 			}
 
-			msg := types.NewMsgRegisterInterchainAccount(connectionID, owner, version)
+			orderString, err := cmd.Flags().GetString(flagOrdering)
+			if err != nil {
+				return err
+			}
+			order, err := getOrder(orderString)
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgRegisterInterchainAccount(connectionID, owner, version, order)
 
 			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
 		},
 	}
 
 	cmd.Flags().String(flagVersion, "", "Controller chain channel version")
+	cmd.Flags().String(flagOrdering, "ORDER_ORDERED", "Channel ordering")
```

Comment:
don't really like this but it does make it slightly easier to get `Order` afterwards. Can do simply `"ORDERED"` and just glue things together if people prefer that.